src/atdf/interrupt.rs

```diff
@@ -0,0 +1,26 @@
+use crate::{
+    chip,
+    chip::Interrupt,
+    util,
+    ElementExt,
+};
```

Comment:
Please don't use brace-imports, and also don't import names (like `Interrupt`) directly.  Instead, reference them later by prefixing their module (`chip::Interrupt`).